plugins/trace_api_plugin/request_handler.cpp

```diff
@@ -25,6 +25,22 @@ namespace {
 
    }
 
+   fc::variants process_call_traces(const std::vector<call_trace_v0>& call_traces) {
+      fc::variants result;
+      result.reserve(call_traces.size());
+      for ( const auto& t: call_traces) {
+         result.emplace_back(fc::mutable_variant_object()
+            ("call_ordinal", t.call_ordinal)
+            ("sender_ordinal", t.sender_ordinal)
+            ("receiver", t.receiver.to_string())
+            ("data", fc::to_hex(t.data.data(), t.data.size()))
+            ("return_value", fc::to_hex(t.return_value.data(), t.return_value.size()))
```

Comment:
Will need to add variant decode support for both `data` and `return_value` when ABI support is available.

Reply:
Yes, I plan to do it when ABI is being worked. Thanks.